html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-001.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>HTML: window.open `features`: tokenization -- `noopener`</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#apis-for-creating-and-navigating-browsing-contexts-by-name">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+var windowURL = 'resources/close-self.html';
+
+test (t => {
+  var featureVariants = [
+    'NOOPENER',
+    'noOpenER'
+  ];
+  featureVariants.forEach(feature => {
+    var win = window.open(windowURL, '', feature);
+    assert_equals(win, null, `"${feature}" should be tokenized -> name "noopener"`);
+  });
+}, 'feature name should be converted to ASCII lowercase');
+
+test (t => {
+  var featureVariants = [
+    '  noopener',
+    'noopener = ',
+    ' no opener',
```

Comment:
I must've misread the steps about ignoring whitespace in the tokenization: is it the case that whitespace inside/within of a `name` is _not_ ignored? Only "trimmed" from start/end, as it were? I saw some [comments in the HTML spec thread](https://github.com/whatwg/html/pull/2476#issuecomment-291129043) about white space and specific handling of characters. That's helpful information! Keep it coming and I'll get the right tests in place.